clang/lib/Analysis/FlowSensitive/DataflowEnvironment.cpp

```diff
@@ -290,13 +295,13 @@ widenKeyToValueMap(const llvm::MapVector<Key, Value *> &CurMap,
 namespace {
 
 // Visitor that builds a map from record prvalues to result objects.
-// This traverses the body of the function to be analyzed; for each result
-// object that it encounters, it propagates the storage location of the result
-// object to all record prvalues that can initialize it.
+// This traverses the AST sub-tree to be visited; for each result object that
```

Comment:
I found "analyzed" to be confusing because it seemed to imply that this Visitor was doing the analyzing, which within Environment typically refers to a dataflow analysis, not any generic analysis of something.

Looks like all starting calls to use this Visitor use a Traverse* method, so the fact that it traverses seems obvious enough that we can just remove this clause.

Reply:
> I found "analyzed" to be confusing because it seemed to imply that this Visitor was doing the analyzing, which within Environment typically refers to a dataflow analysis, not any generic analysis of something.

Ah, understood. What I meant when I wrote the original comment was indeed that the visitor traverses the body of the function on which we later want to perform dataflow analysis. But I see how this can be confusing.

Agree that this bit can simply be dropped; "this traverses the AST sub-tree to be visited" is true of any `RecursiveASTVisitor`.